### ISSUE TITLE
v1 failed with the caller does not have permission

### ISSUE THREAD

**Issue by avalero** [google-github-actions/get-gke-credentials#260]
### TL;DR

Running the example of the tutorial fails with: `google-github-actions/get-gke-credentials failed with: the caller does not have permission`

### Expected behavior

It should get te credentials

### Observed behavior

It outputs the error: google-github-actions/get-gke-credentials failed with: the caller does not have permission

### Action YAML

```yaml
name: front at prod

on:
  workflow_dispatch:
  release:
    types:
      - created

# Environment variables available to all jobs and steps in this workflow
env:
  GKE_PROJECT: ${{ secrets.GKE_PROJECT }}
  GKE_EMAIL: ${{ secrets.GKE_EMAIL }}
  GKE_IDENTITY_PROVDER: ${{ secrets.GKE_IDENTITY_PROVIDER }}
  GITHUB_SHA: ${{ github.sha }}
  GITHUB_REF: ${{ github.ref }}
  GKE_ZONE: europe-west1
  GKE_CLUSTER: ****
  IMAGE: *****
  REGISTRY_HOSTNAME: eu.gcr.io
  DEPLOYMENT_NAME: front
  NAMESPACE: prod
  SERVICE: *****

jobs:
  frontend:
    permissions:
      contents: 'read'
      id-token: 'write'
    name: Setup, Build, Publish, and Deploy frontend at prod
    runs-on: ubuntu-latest
    steps:
      # Checkout
      - name: Checkout
        uses: actions/checkout@v3

      # Setup gcloud CLI

      - id: "auth"
        name: 'Authenticate to Google Cloud'
        uses: 'google-github-actions/auth@v1'
        with:
          service_account: ${{ secrets.GKE_EMAIL }}
          workload_identity_provider: ${{ secrets.GKE_IDENTITY_PROVIDER }}
      - id: 'get-credentials'
        uses: 'google-github-actions/get-gke-credentials@v1'
        with:
          cluster_name: $GKE_CLUSTER
          location: $GKE_ZONE

      # Configure docker to use the gcloud command-line tool as a credential helper
      - name: Configure docker
        run: |
          # Set up docker to authenticate
          # via gcloud command-line tool.
          gcloud auth configure-docker
      #Copy env.dev
      - name: Copy env -> .env
        run: cp ./packages/frontend/.env.$NAMESPACE ./packages/frontend/.env
        shell: bash
        continue-on-error: true

      # Build the Docker image
      - name: Build Frontend App
        run: |
          docker build -f ./packages/frontend/Dockerfile -t $REGISTRY_HOSTNAME/$GKE_PROJECT/$IMAGE:$DEPLOYMENT_NAME-$GITHUB_SHA .
      # Push the Docker image to Google Container Registry
      - name: Publish
        run: |
          docker push $REGISTRY_HOSTNAME/$GKE_PROJECT/$IMAGE:$DEPLOYMENT_NAME-$GITHUB_SHA
      - name: "Deploy to Cloud Run"
        uses: "google-github-actions/deploy-cloudrun@v0"
        with:
          project_id: ${{ secrets.GKE_PROJECT }}
          service: "${{ env.SERVICE }}"
          image: "${{ env.REGISTRY_HOSTNAME  }}/${{ env.GKE_PROJECT }}/${{ env.IMAGE }}:${{ env.DEPLOYMENT_NAME }}-${{ env.GITHUB_SHA }}"
          region: "${{ env.GKE_ZONE }}"
          env_vars: 'SHA="${{ env.GITHUB_SHA }}"'

      # Google chat notification
      - name: Google Chat Notification
        uses: Co-qn/google-chat-notification@releases/v1
        with:
          name: Bitbloq PROD - Frontend build
          url: ${{ secrets.GOOGLE_CHAT_WEBHOOK }}
          status: ${{ job.status }}
        if: always()
```
```


### Log output

```text
Run google-github-actions/get-gke-credentials@v1
Extracted project ID "***" from $GCLOUD_PROJECT
Error: google-github-actions/get-gke-credentials failed with: the caller does not have permission
```

### Additional information

service accounts has these permissions

Cloud Run Admin
GKE Hub Viewer
Kubernetes Engine Admin
Kubernetes Engine Cluster Viewer
Service Account Token Creator
Service Account User
Storage Admin
Storage Object Admin

**Comment by sethvargo** [comment]
Hi @avalero - please post the [debug logs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging)

**Comment by avalero** [comment]
Hello @sethvargo , I have found the problem, the repo name is case sensitive, while going the github url is not. Changing the repo case (to the correct value) it works.
Sorry and thankyou for your time.
